epsagon_kubernetes_subscription.sh

```diff
@@ -144,9 +144,12 @@ function apply_role {
         else
             ROLE_TOKEN=`${KUBECTL} -n epsagon-monitoring get secrets $SA_SECRET_NAME -o json | grep '\"token\"' | cut -d: -f2 | cut -d'"' -f2 | base64 --decode`
         fi
-        apply_mutation_controller $EPSAGON_TOKEN $CONFIG $CONTEXT
-
-        send_to_epsagon $EPSAGON_TOKEN $ROLE_TOKEN $CONFIG $CONTEXT
+        if [ -z $ROLE_TOKEN ]; then
+            echo "Role creation failed, could not get role token"
```

Comment:
I would change the error message to something else, and exit with status code != 0

Reply:
I don't want to exit because the user might have other clusters that we will be able to create a role for (e.g. if role creation failed because this specific cluster is not reachable at the moment). 
What error message would you suggest?

Reply:
Yes you're right.
An error message that will indicate its a deployment problem on that cluster, maybe: Failed to deploy epsagon role on chosen cluster - couldn't extract role token (something like that)